.github/workflows/ci.yaml

```
@@ -0,0 +1,22 @@
+name: test
```

I think this name should be kept the same
```suggestion
name: build
```

---

Yeah.. I was trying to be consistent with https://github.com/rancher/lasso/pull/68/files#diff-944291df2c9c06359d37cc8833d182d705c9e8c3108e7cfe132d61a06e9133dd but that's not merged yet. 

I can change it to CI instead, to be more consistent with https://github.com/rancher/dynamiclistener/blob/master/.github/workflows/ci.yaml. How about that? Might make more sense too since that job runs many steps:
- validation
- test
- build

WDYT?

---

I like calling it CI. If we ever move away from dapper and split the CI we can name the steps appropriately then